packages/data-point/lib/reducer-object/resolve.test.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-env jest */
```

Comment:
if by any chance there is an error on any of the promises here it will not be caught. I would recommend use the `.catch(e => e)` after each promise execution, to make sure your error is not swallowed by the promises. 

In theory, Jest should catch the error and warn/error out. but has been my experience that it doesn't and I have ended up with a set of tests that never noticed had errors.

Reply:
disregard, just checked, all good here...